wof/WaterML_1_1.py

```diff
@@ -405,7 +405,7 @@ def quote_xml(inStr):
 
 
 def quote_xml_aux(inStr):
-    inStr = inStr.decode('utf-8')
+    inStr = inStr.encode('utf-8').decode('utf-8')
```

Comment:
I am never really comfortable with this b/c, IMO, we should always treat incoming data as bytes, but there is no other way around this unless we start supporting Python 3.

TL;DR :+1: